pkg/cursor/cursor_test.go

```diff
@@ -0,0 +1,77 @@
+package cursor
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_Comparable(t *testing.T) {
+	tests := []struct {
+		name    string
+		c1      string
+		c1Error bool
+		c2      string
+		c2Error bool
+
+		isComparable bool
+		isEqual      bool
+		isBefore     bool
+		isAfter      bool
+	}{
+		{
+			name:         "comparable and equal",
+			c1:           "10",
+			c2:           "10",
+			isEqual:      true,
+			isComparable: true,
+		},
+		{
+			name:         "comparable and before",
+			c1:           "1",
+			c2:           "10",
+			isBefore:     true,
+			isComparable: true,
+		},
+		{
+			name:         "comparable and after",
+			c1:           "100",
+			c2:           "10",
+			isAfter:      true,
+			isComparable: true,
+		},
+		{
+			name:    "not comparable",
```

Comment:
this isn't testing the "not comparable" function though? Just whether C1 forms a cursor properly